google/cloud/storage/bucket.py

```diff
@@ -387,7 +387,7 @@ class IAMConfiguration(dict):
 
     :type public_access_prevention: str
     :params public_access_prevention:
-        (Optional) Whether the public access prevention policy is 'unspecified' (default) or 'enforced'
+        (Optional) Whether the public access prevention policy is 'inherited' (default) or 'enforced'
         See: https://cloud.google.com/storage/docs/public-access-prevention
```

Comment:
Duplicate line.

Reply:
Hm? I don't follow, this one was there, it just changed the label.

Reply:
Line 392 is a duplicate of 391. You didn't duplicate it -- it was that way already.

Reply:
Ohh I was looking at 390, woops. Good catch